en_US/data-integration/snowflake.md

```diff
@@ -385,60 +380,72 @@ Before configuring the Snowflake Sink, you must create a connector in EMQX to es
+
      - **Proxy Host**: The hostname or IP address of the proxy server.
      - **Proxy Port**: The port number used by the proxy server.
+   
+6. If you want to establish an encrypted connection, click the **Enable TLS** toggle switch. For more information about TLS connection, see [TLS for External Resource Access](../network/overview.md/#tls-for-external-resource-access). TLS must be enabled for streaming mode, as communication is over HTTPS.
 
-   :::
+7. Advanced settings (optional): See [Advanced Settings](#advanced-settings).
+
+8. Before clicking **Create**, you can click **Test Connectivity** to test if the connector can connect to the Snowflake.
+
+9. Click the **Create** button at the bottom to complete the connector creation.
+
+You have now completed the connector creation and can proceed to create a rule and Sink to specify how the data will be written into Snowflake.
+
+## Create a Snowflake Streaming Connector
+
+If you plan to use the streaming upload mode in your Snowflake Sink, you need to create a Snowflake Streaming Connector to establish the connection with your Snowflake environment. This connector uses HTTPS and the Snowpipe Streaming REST API (AWS-only).
+
+1. Go to the Dashboard **Integration** -> **Connector** page.
+
+2. Click the **Create** button in the top right corner.
+
+3. Select **Snowflake Streaming** as the connector type and click next.
+
+4. Enter the connector name, a combination of upper and lowercase letters and numbers. Here, enter `my-snowflake-streaming`.
+
+5. Enter the connection information.
 
-   ::: tab Streaming Mode (Snowpipe Streaming API)
-   
-   - **Account**: Enter your Snowflake Organization ID and Snowflake account name separated by a dash (`-`), which is part of the URL you use to access the Snowflake platform and can be found in your Snowflake console.
    - **Server Host**: The server host is the Snowflake endpoint URL, typically in the format `<Your Snowflake Organization ID>-<Your Snowflake Account Name>.snowflakecomputing.com`. You need to replace `<Your Snowflake Organization ID>-<Your Snowflake Account Name>` with the subdomain specific to your Snowflake instance.
-   - **Username**: (Optional) Enter the Snowflake user with a registered RSA public key if you configured it in `odbc.ini` (e.g. `snowpipeuser`). 
+   - **Account**: Enter your Snowflake Organization ID and Snowflake account name separated by a dash (`-`), which is part of the URL you use to access the Snowflake platform and can be found in your Snowflake console.
+   - **Pipe User**: The name of a Snowflake user account that has a role with permissions to operate the target Pipe, for example, `snowpipeuser`. The role must have at least the `OPERATE` and `MONITOR` privileges.
    - **Private Key Path**: The absolute file path to the private RSA key. EMQX uses this key to sign JWT tokens to authenticate itself with the Snowflake API. This path must be consistent across all nodes of the cluster. For example:
```

Comment:
For the streaming connector, this is no longer a cluster-wide path but a regular secret (i.e., can be specified inline, or point to a file containing the secret with `file://`)